core/queryalgebra/evaluation/src/main/java/org/eclipse/rdf4j/query/algebra/evaluation/iterator/ProjectionIterator.java

```diff
@@ -1,12 +1,16 @@
-/*******************************************************************************
+/** *****************************************************************************
```

Comment:
Looks like your code formatter mangles the copyright header a bit?

Reply:
fixed